pkg/apis/performanceprofile/v2/performanceprofile_validation_test.go

```diff
@@ -647,6 +651,59 @@ var _ = Describe("PerformanceProfile", func() {
 			Expect(errors).ToNot(BeEmpty())
 			Expect(errors[0].Error()).To(ContainSubstring(("the page size should be equal to one of")))
 		})
+		Context("KernelPageSize with HugePages validation", func() {
+			var kernelToHugePageSizeByArch = map[string]map[string][]string{
+				aarch64: aarch64HugePagesByKernelPageSize,
+				amd64:   x86HugePagesByKernelPageSize,
+			}
+
+			var tableEntries []TableEntry
+			allHugePageSizes := allHugePageSizes()
```

Comment:
try being consistent with other values above (using set suffix if all should use one for example)